IdentityCore/src/intune/MSIDIntuneEnrollmentIdsCache.m

```diff
@@ -193,6 +193,11 @@ - (NSDictionary *)enrollmentIdsJsonDictionaryWithContext:(id<MSIDRequestContext>
     return jsonDictionary;
 }
 
+- (void)clear
+{
+    [self.dataSource removeObjectForKey:MSID_INTUNE_ENROLLMENT_ID_KEY];
```

Comment:
is this thread safe?

Reply:
Our data sources are MSIDCache & NSUserDefaults, both are thread-safe.